hbp_knowledge/receptors/wu2011.bel

```diff
@@ -75,60 +75,84 @@ SET Citation = {"PubMed", "21787755"}
 SET Evidence = "Moreover, some nAChRs have been implicated in processes such as
 the structuring and maintenance of neurites and synapses [18–20]
 and even in modulation of neuronal viability/death [21–24]."
+SET Confidence = "High"
 p(FPLX:CHRN) -> bp(GO:"maintenance of synapse structure")
 p(FPLX:CHRN) -> bp(GO:"neuron projection maintenance")
+UNSET Confidence
 
 SET Evidence = "nAChRs that bind radiolabeled nicotine with the highest afﬁnity
 contain a4 subunits (a4*-nAChR; see reviews and/or tables by [1–7]."
-complex(a(HBP:"alpha-4-containing nAChR"),a(CHEBI:nicotine))
+SET Confidence = "Medium"
+composite(a(HBP:"alpha-4-containing nAChR"),a(CHEBI:nicotine)) -> complex(a(HBP:"alpha-4-containing nAChR"),a(CHEBI:nicotine))
+UNSET Confidence
 
 SET Evidence = " a4b2-nAChRs have been
 implicated in nicotine self-administration, reward, and depen-
 dence, and in diseases such as Alzheimer’s and epilepsy [1–5,27–33]."
-a(HBP:"AβOs") -- path(MESH:"Tobacco Use Disorder")
-a(HBP:"AβOs") -- path(MESH:"Alzheimer Disease")
-a(HBP:"AβOs") -- path(MESH:Epilepsy)
+SET Confidence = "Medium"
+complex(p(HGNC:CHRNA4),p(HGNC:CHRNB2)) -- path(MESH:"Tobacco Use Disorder")
+complex(p(HGNC:CHRNA4),p(HGNC:CHRNB2)) -- path(MESH:Reward)
+complex(p(HGNC:CHRNA4),p(HGNC:CHRNB2)) -- path(MESH:"Alzheimer Disease")
+complex(p(HGNC:CHRNA4),p(HGNC:CHRNB2)) -- path(MESH:Epilepsy)
+# ABOs to complex(p(HGNC:CHRNA4),p(HGNC:CHRNB2))
+UNSET Confidence
 
 SET Evidence = "a4 subunits can also assemble with b4 subunits to form
 a4b4-nAChRs that have comparably high nicotine afﬁnity [34–36]."
-complex(a(HBP:"alpha-4 beta-4 nAChR"),a(CHEBI:nicotine))
+SET Confidence = "Medium"
+composite(p(HGNC:CHRNA4),p(HGNC:CHRNB4)) -> a(HBP:"alpha-4 beta-4 nAChR")
```

Comment:
I think you can also go with the isA term for this ( or is there an other way to encode that they are equal? )

Reply:
The previous sentence on complex formation of A4B4 nAChR suited much better.